src/dropdownButton/index.tsx

```diff
@@ -82,7 +82,7 @@ class DropdownButton extends PureComponent <DropdownButtonProps, DropdownButtonS
 
     return (
       <div ref={this.refButton} className="dropdownButton">
-        <button className={cc(className)} onClick={this.toggle}>
+        <button type="button" className={cc(className)} onClick={this.toggle}>
```

Comment:
Can you also add a unit test in `/src/dropdownButton/index.unit.tsx`?
Something like this:
 ```
it('Should have an attribute type button', () => {
    const wrapper = shallow(<DropdownButton {...defaultProps} />)
    expect(wrapper.find('button[type="button"]').toHaveLength(1)
  })

```

Reply:
Done